app/assets/javascripts/spreadsheet.js

```diff
@@ -1,58 +1,67 @@
 // This file contains JavaScript specific functionality for the bulk loading of spreadsheets.
 
-function enable_submit()
-{
-    $("#spreadsheet_submit").prop("disabled", false);
-}
-
-function disable_control(ctrl)
-{
-    ctrl.prop("disabled", true)
-}
-
-function enable_control(ctrl)
-{
-    ctrl.prop("disabled", false)
-}
-
-
-// The Blacklight onLoad event works better than the regular onLoad event if turbolinks is enabled.
-Blacklight.onLoad(function(){
+// An object to prevent us from polluting the global namespace with all these generically named functions.
+var argoSpreadsheet = {
+    
+    enable_submit : function()
+    {
+	$("#spreadsheet_submit").prop("disabled", false);
+    },
+    
+    disable_control : function(ctrl)
+    {
+	ctrl.prop("disabled", true);
+    },
+    
+    enable_control : function(ctrl)
+    {
+	ctrl.prop("disabled", false);
+    },
 
-    // When the user clicks the "MODS bulk loads" button, a lightbox is opened. The event
-    // "loaded.blacklight.ajax-modal" is fired just before this Blacklight lightbox is shown.
-    $("#ajax-modal").on("loaded.blacklight.ajax-modal", function(e){
-	var input_controls = [$("#filetypes_1"), $("#filetypes_2"), $("#convert_only"), $("#note_text")]
-	var radio_buttons = input_controls.slice(0, 3)
+    load_upload_form : function()
+    {
+	// When the user clicks the "MODS bulk loads" button, a lightbox is opened. The event
+	// "loaded.blacklight.ajax-modal" is fired just before this Blacklight lightbox is shown.
+	var input_controls = [$("#filetypes_1"), $("#filetypes_2"), $("#convert_only"), $("#note_text")];
```

Comment:
Line is too long.<br>Missing "use strict" statement.<br>Identifier 'input_controls' is not in camel case.<br>Strings must use singlequote.